spec/features/organizations/user_edits_organization_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+feature "org_admin updates organization" do
+
+  let!(:user) { FactoryGirl.create(:user, volunteer: false) }
+  let!(:organization) { FactoryGirl.create(:organization) }
+  let!(:org_admin) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.